sprout/core/verify_is_directory.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+
+def verify_is_directory(path: Path) -> Path:
+    """Verifies if the path is an existing directory.
+
+    Args:
+        path: The path to verify
+
+    Raises:
+        NotADirectoryError: If the path is not an existing directory
```

Comment:
```suggestion
    Raises:
        NotADirectoryError: When the directory in the path doesn't exist.
```

Just some small grammar/language change.